### DISCUSSION TITLE
How to update a package only if the package gets updated in different directory

### DISCUSSION THREAD

**Discussion by batbattur** [renovatebot/renovate#26584] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.128.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

How to update a package only if the same package is updated in a different folder?

For example, I have the following structure.
- `app/package.json`
- `git-submodule-dir/package.json`

Say I have package `A` in both `app/package.json` and `git-submodule-dir/package.json`. 

I want package `A` updated only if package "A" gets updated in `git-submodule-dir/package.json`.

I checked the [packageRules](https://docs.renovatebot.com/configuration-options/#packagerules) config but didn't find that can achieve this.

**Comment by rarkins** [comment, marked as answer]
No such functionality exists. Maybe you can play around with minimumReleaseAge to delay the PR enough that both are available at the same time
